components/tabs/ProductsRequestsTabs.tsx

```diff
@@ -13,22 +13,22 @@ export default function Tabs() {
         type="button"
         href={`/${pathname.split('/')[1]}/products/all`}
         className={`pl-4 relative inline-flex items-center rounded-l-lg px-3 py-2 text-sm font-semibold ring-1 ring-inset ring-bcblue focus:z-10
-          ${
-            pathname.split('/')[2] === 'products' ? 'bg-bcblue text-white' : 'bg-white text-gray-900 hover:bg-gray-100'
-          }`}
+          ${pathname.split('/')[3] === 'all' ? 'bg-bcblue text-white' : 'bg-white text-gray-900 hover:bg-gray-100'}`}
       >
         Products
       </Link>
       <Link
-        style={{ width: 97 }}
+        style={{ width: 114 }}
```

Comment:
Can the `width` be applied as in a `tailwind class`?

Reply:
Yes I will make that tailwind for consistency.